bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/dataexchange/NukiApiServlet.java

```diff
@@ -161,6 +162,11 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
                         State state = bridgeApiLockStateRequestDto.isBatteryCritical() ? OnOffType.ON : OnOffType.OFF;
                         nsh.handleApiServletUpdate(channel.getUID(), state);
                     }
+                    channel = thing.getChannel(NukiBindingConstants.CHANNEL_SMARTLOCK_DOOR_STATE);
+                    if (channel != null) {
+                        State state = new DecimalType(bridgeApiLockStateRequestDto.getDoorsensorState());
```

Comment:
See other comment about channel item-type:
```suggestion
                        State state = new StringType(bridgeApiLockStateRequestDto.getDoorsensorState());
```